src/core/unit.c

```diff
@@ -4012,7 +4012,7 @@ void unit_warn_if_dir_nonempty(Unit *u, const char* where) {
         assert(where);
 
         r = dir_is_empty(where);
-        if (r > 0)
+        if ((r > 0) || (r == -ENOTDIR))
                 return;
```

Comment:
Extra `()` are not necessary.

Reply:
Surely it is, I just find it easier to read. But If this is a part of the coding style I'll push a new version.

Reply:
No no... I mean `if (r > 0 || r == -ENOTDIR)`.